src/entities/trades/curve/CurveTrade.ts

```diff
@@ -220,26 +297,16 @@ export class CurveTrade extends Trade {
           })
         : undefined
 
-    // Majority of Curve pools
-    // have 4bps fee of which 50% goes to Curve
-    const FEE_DECIMAL = 0.0004
-    let fee = new Percent('4', '10000')
-
-    // Exchange fee
-    const exchangeRateWithoutFee = 1
-    const exchangeRate = 1 - FEE_DECIMAL
-
     // If a pool is found
     // Ignore the manual off-chain search
     if (bestPoolAndOutputRes) {
-      debug(`Curve::GetQuote | Found best pool from Curve`, bestPoolAndOutputRes)
+      debugCurveGetQuote(`Found best pool from Curve registery`, bestPoolAndOutputRes)
```

Comment:
I think there is a typo

```suggestion
      debugCurveGetQuote(`Found best pool from Curve registry`, bestPoolAndOutputRes)
```

Reply:
Addressed in https://github.com/levelkdev/swapr-sdk/pull/80/commits/55595d108ce509867f961b739bfea556f9cc792f